recipes-domd/domd-image-weston/domd-image-weston.bbappend

```diff
@@ -175,6 +175,13 @@ configure_versions_rcar() {
     fi
 
     base_update_conf_value ${local_conf} XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR "${XT_RCAR_PROPRIETARY_MULTIMEDIA_DIR}"
+
+    # We have netevent_%.bbappend that modifies
+    # meta-rcar-gen3-adas/recipes-support/netevent/netevent_git.bb
+    # and is intended to be used for Kingfisher only.
+    # For other boards we need to mask our bbappend to avoid
+    # warning "No recipes available for".
+    base_add_conf_value ${local_conf} BBMASK "recipes-support/netevent"
```

Comment:
This is a good change. But it should be made in a separate commit.

Reply:
Done. BBMASK is moved to separate commit.